### DISCUSSION TITLE
Player and Chat Custom Position and Size

### DISCUSSION THREAD

**Discussion by sunyoung1** [streamlink/streamlink#4587] [Q&A] [answered]
Hi, just wanna ask if it is possible to set custom window size and position of the player and chat by default since it's irritating to resize chat window and player every single time i open a stream.
I tried fiddling with default chrome settings to start up with my custom window size and position and it does work when i open chrome. It doesn't work when being used as a chat app for streamlink. Also wanna know if this is possible with player.

**Comment by gravyboat** [comment]
It's not. We don't have anything to do with the management of whatever chat application you choose to use. You could attempt to write some sort of wrapper script that does the window position sizing and so forth but that isn't something we can provide assistance with.

**Comment by mkbloke** [comment, marked as answer]
You don't say what OS or player you're using.  Under Linux with MPV, you can set the window geometry with `--geometry` in the player arguments specified with `-a`:
```
streamlink -p mpv -a '--geometry=+0+0' URL QUALITY
```
That will place the window in the top left corner.  You can also set the width and height using the same option.  See the MPV docs/manual page for more information.  VLC may also have command line options to do the same thing, but you'll have to look those up yourself.
